pkgs/data/fonts/blobmoji/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cairo, pngquant, zopfli
+, which, pythonPackages, pkgconfig, imagemagick }: stdenv.mkDerivation rec {
+  pname = "blobmoji";
+  version = "2019-06-14-Emoji-12";
+
+  src = fetchFromGitHub {
+    owner = "C1710";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "19xwjarpn40jyibp7d2g0ss2c9hifrskkig234wdng08gc1p4z84";
+  };
+
+  buildInputs = [ cairo ];
+  nativeBuildInputs = [ cairo pngquant zopfli which pkgconfig imagemagick ]
+                      ++ (with pythonPackages; [ python fonttools nototools ]);
+  enableParallelBuilding = true;
+
+  postPatch = ''
+    sed -i 's,^PNGQUANT :=.*,PNGQUANT := ${pngquant}/bin/pngquant,' Makefile
+    patchShebangs flag_glyph_name.py
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/noto
+    cp NotoColorEmoji.ttf fonts/Blobmoji.ttf $out/share/fonts/noto
```

Comment:
Should it be copying `NotoColorEmoji.ttf` to the output? Both files have the font name "Blobmoji"

```
[nix-shell:~/tmp/nixpkgs/nixpkgs]$ ls -lh result/share/fonts/noto/
total 16M
-r--r--r-- 1 root root 8.0M Dec 31  1969 Blobmoji.ttf
-r--r--r-- 1 root root 7.4M Dec 31  1969 NotoColorEmoji.ttf
```

I am concerned in two things

 * `NotoColorEmoji.ttf` clashing with noto color emoji.
 * Both font files fighting for the Blobmoji font name.

It seems that [releases only include Blobmoji.ttf](https://github.com/C1710/blobmoji/releases). That file being bigger also makes me think the NotoColorEmoji is a transient build artifact. Though, I wasn't able to figure it quickly.

Reply:
I'm aware of the potential conflict. But as blobmoji is just noto color with additional blobby emoji so it's unlikely somebody would use both at the same time.

Reply:
The thing is the filename is `NotoColorEmoji`, but the font name is `Blobmoji Regular`. I think (though I'm not sure) that this is an issue since both font files will respond to the `Blobmoji Regular` font name.

That is more confusing with the conflict with the presence of `NotoColorEmoji.ttf` file. The conflict breaks the assumption that the noto color emoji font (by name) is made available.

I'm not sure if either of @jtojnar or @worldofpeace would know more about issues this could cause (they recently worked on color emoji stuff).

Reply:
Fontconfig chooses a font based on distance from a property vector. File name is [one of the matched properties](https://www.freedesktop.org/software/fontconfig/fontconfig-user.html#AEN21) but I doubt either of them would win based on that, unless the application explicitly asked for that file name. Not sure what happens when both have the same distance, would need to see the files. On the other hand, if `NotoColorEmoji.ttf` fits better, it could definitely win.